test/CMakeLists.txt

```diff
@@ -148,15 +148,15 @@ teca_add_test(test_descriptive_statistics_threads
 
 teca_add_test(test_descriptive_statistics_mpi
     LIBS teca_core teca_data teca_io teca_alg ${teca_test_link}
-    COMMAND ${MPIEXEC} -n ${TECA_TEST_CORES} test_descriptive_statistics
+    COMMAND ${MPIEXEC} -n ${TECA_TEST_CORES} --oversubscribe test_descriptive_statistics
```

Comment:
I think this is an open-mpi specific option, and it may not work to have it hard coded. But before going there, a question.

how many cores are there on these systems? Assuming its not a 1 core system, we might want avoid over subscription. 

You can control the number of cores the tests run over by adding `-DTECA_TEST_CORES=N` where N > 1 to the cmake command line. In the current setup we had been using `TECA_TEST_CORES=2`, this is set in `test/travis_ci/ctest+linux.cmake`